### PR TITLE
23 allow passing a lambda as record

### LINKED ISSUE #23: Allow passing a lambda as record
This should be passed the usual root, arguments, context

### PR DESCRIPTION
**Pull request by phyrog**
Closes #23.

### CODE REVIEW

File: lib/graphql-pundit/instrumenters/authorization.rb [hunk 1]
```diff
@@ -25,15 +25,21 @@ def instrument(_type, field)
         end
 
         # rubocop:disable Metrics/MethodLength, Metrics/AbcSize
+        # rubocop:disable Metrics/CyclomaticComplexity
+        # rubocop:disable Metrics/PerceivedComplexity
+        # rubocop:disable Metrics/BlockLength
         def resolve_proc(current_user, old_resolve, options)
-          # rubocop:enable Metrics/MethodLength, Metrics/AbcSize
```
Comment (eugenk) [inline lib/graphql-pundit/instrumenters/authorization.rb]: Why is this line removed?

File: spec/graphql-pundit/instrumenters/authorization_spec.rb [hunk 2]
```diff
@@ -1,6 +1,7 @@
 # frozen_string_literal: true
 
 require 'spec_helper'
+require 'pry'
```
Comment (eugenk) [inline spec/graphql-pundit/instrumenters/authorization_spec.rb]: Seems to be a leftover from debugging.

File: lib/graphql-pundit/instrumenters/authorization.rb (hunk 1 above)
Comment (phyrog) [reply lib/graphql-pundit/instrumenters/authorization.rb]: It's the last method in the file and re-enabling all disabled cops would add a lot of clutter. I'm planning on refactoring this file after this PR (or in this PR if you'd like).

File: spec/graphql-pundit/instrumenters/authorization_spec.rb (hunk 2 above)
Comment (phyrog) [reply spec/graphql-pundit/instrumenters/authorization_spec.rb]: Fixed in rebase
